README.md

```diff
@@ -8,114 +8,109 @@ Cellotape requires Go 1.18 or above.
-
-  Validated components includes:
-
-  - **Request Body Schema**
-
-  - **Path Parameters**
-
-  - **Query Parameters**
-
-  - **Responses**
-
-- Support for middleware chains and group mechanism that allow applying middlewares to specific operations or specific groups.
-
-- Compatability with the `http.Handler` interface for both the router itself and the middlewares to allow easy integration of the router in any popular framework.
-
+- Load, parse, and validate an OpenAPI spec (`router.OpenAPISpec`).
+- Initialize an HTTP router driven from an OpenAPI spec (`router.OpenAPIRouter`).
+- An SDK that defines strongly typed handlers mapped to spec operations.
+- Verify handler signature compatibility with spec operations.
+  During initialization, this enforces that the handlers correctly implement the 
+  spec. The validated components include:
+  - Request body schema
+  - Path parameters
+  - Query parameters
+  - Responses
+- Support for middleware chain and group mechanisms that enable middleware to 
+  apply specific operations or specific groups.
+- Compatibility with the `HTTP.Handler` interface for the router and middleware 
+  to enable easy integration of the router into any popular framework.
 - Support for custom content types to align with content types defined in the spec.
-
-  This can be done by implementing the `router.ContentType` interface
-
-- Support for customization of validation behaviour and other configuration using `router.Options`.
-
-  For the full documentation of the available options you can check the documentation on the [`router.Options`](./router/options.go) struct.
-
+  This can be done by implementing the `router.ContentType` interface.
+- Support for customization of validation behavior and other configuration using
+  `router.Options`.
+  See the documentation on the [`router.Options`](./router/options.Go) struct 
+  for details of the options.
 
 ### What this project isn't doing
 
 - It's not a code generator.
+  Code generators have their issues, such as minimal control over the generated 
+  code and the overwriting of manual edits whenever the spec changes.
 
-  Code generators have their issues, with minimal control over the generated code, and the difficulty to edit generated code that can overridden whenever the spec changes.
-
-- It's not using comments on your code. Some approaches might suggest you comment your code in a way that produce an OpenAPI spec that can be compared to the spec you design.
-
-  The issue with this approach is that the spec, code and comments can get out of sync too easily.
+- It's not using comments on your code. Some approaches might suggest you 
+  comment code in a way that produces an OpenAPI spec that can be compared to the 
+  spec.
+  The issue with this approach is that the spec, code, and comments can get out 
+  of sync too easily. Primarily when HTTP handlers in Go provide an opaque 
+  interface with no type information.
```

Comment:
I think sentence is still a bit unclear (it wasn't so clear also before).

```suggestion
- It's not using comments on your code. Some approaches might suggest you 
  comment code in a way that produces an OpenAPI spec (a code first approach).
  
  Then, in order to use a design first approach the produced spec can be compared to the 
  desired spec.
  
  The issue with this approach is that the spec, code, and comments can get out 
  of sync too easily. Primarily when HTTP handlers in Go provide an opaque 
  interface with no type information.
```

Reply:
Suggested a further improvement in the update